### ISSUE TITLE
`asdf current` output is not correct for tools that are not installed

### ISSUE THREAD

**Issue by Stratus3D** [asdf-vm/asdf#528]
### Steps to reproduce
Run `asdf current` in a directory with a `.tool-versions` file containing versions that are not installed. Output will look like this:

```
version 1.8.2-otp-21 is not installed for elixir
elixir         
erlang         21.3.8.2 (set by ~/devel/dir/.tool-versions)
version 11.9.0 is not installed for nodejs
nodejs         
```

### Expected behavior

For all tools listed, both the name and the version should be displayed on one line just like the installed versions. e.g.

```
elixir    version 1.8.2-otp-21 is not installed for elixir
```

### Actual behavior

For tools with versions that are not installed, the message for the tool is shown on a separate line.

```
version 1.8.2-otp-21 is not installed for elixir
elixir
```

### Environment

**OS**: OSX 10.14.3

**asdf version**: v0.7.2

**Comment by jthegedus** [comment]
This issue is fixed for me in `0.7.8`:

```
➜ asdf current
elixir         No version set for elixir; please run `asdf <global | local> elixir <version>`
firebase       8.0.2    (set by /home/jthegedus/.tool-versions)
gcloud         288.0.0  (set by /home/jthegedus/.tool-versions)
gradle         6.3      (set by /home/jthegedus/.tool-versions)
hadolint       v1.17.5  (set by /home/jthegedus/.tool-versions)
java           adopt-openjdk-11.0.6+10 (set by /home/jthegedus/.tool-versions)
maven          3.6.3    (set by /home/jthegedus/.tool-versions)
nodejs         12.16.1  (set by /home/jthegedus/projects/docsify-select/.tool-versions)
ocaml          4.10.0   (set by /home/jthegedus/.tool-versions)
python         3.8.2    (set by /home/jthegedus/.tool-versions)
shellcheck     0.7.1    (set by /home/jthegedus/.tool-versions)
terraform      0.12.24  (set by /home/jthegedus/.tool-versions)
```

Or am I misunderstanding the problem?

**Comment by Stratus3D** [comment]
This is still a problem. This issue exists for tool versions that have been set in a `.tool-versions` file but have not be installed. I tested on my OSX machine just now and it is still a problem. After setting a version of Elixir that I do not yet have installed I see:

```
clojure        No version set for clojure; please run `asdf <global | local> clojure <version>`
version 1.10.0 is not installed for elixir
elixir         
erlang         22.1.5   (set by /Users/<redacted>/.tool-versions)
```

Your output shows the inverse - tools present but no version is set in the `.tool-versions` file.
